components/kubeflow/kfserving/sample-pipeline.py

```diff
@@ -28,7 +28,11 @@ def kfservingPipeline(
     canary_model_uri='gs://kfserving-samples/models/tensorflow/flowers',
```

Comment:
if we keep, shall be flowers-2 for canary

Reply:
Done